app-backend/api/src/main/scala/Router.scala

```diff
@@ -62,6 +63,7 @@ trait Router extends HealthCheckRoutes
     } ~
     pathPrefix("api") {
       pathPrefix("projects") { projectRoutes } ~
+      pathPrefix("aoi") { aoiRoutes } ~
```

Comment:
A point could be made against the length of `areas-of-interest`, but I agree that the meaning of `aoi` is unclear until you look it up. I think it's the only route that is an abbreviation.